wiki/How_You_Can_Help!/en.md

```diff
@@ -1,80 +1,78 @@
 
-## Help answer technical, beatmap and gameplay support questions!
+- Choose a forum that sparks your interest:
+   - [Technical Support](/community/forums/5)
+   - [Beatmapping Support](/community/forums/56)
+   - [Storyboarding](/community/forums/20)
+   - [Gamplay and Rankings](/community/forums/13)
 
-Every day I try to take care of any outstanding tech support questions, but a lot of these can be answered by anyone who has spend time around here and knows how things generally work. Even if you don't think you know stuff well enough, you can follow along, read and learn, then start helping at a later point!
+- Mark the forum as read using the link at the top right.
+- Follow any new threads, reading each one and helping out when you can. You can use the unread icons to make sure you keep up.
+- Read the [available guides](/wiki/List_of_Guides "List of Guides") to polish up your osu! knowledge. If the guides in there doesn't satisfy you, you can contribute to them!
 
-### Where to start:
+## Help check/mod beatmaps!
 
--   Choose a forum: [Technical Support](/community/forums/5), [Beatmapping Help](/community/forums/56), [Storyboarding](/community/forums/20) or [Gameplay and Rankings](/community/forums/13) based on what you find most interesting.
--   Mark the forum as read using the link at the top-right.
--   Follow any new threads, reading each one and helping out where you can. You can use the unread icons to make sure you keep up!
--   Read the [available guides](/wiki/List_of_Guides "List of Guides") and polish up on your osu! knowledge. Hell, contribute to the guides if you can!
+osu! has a dedicated team of reviewers called "modders" to check if the beatmaps are ready to be ranked and played by everyone. However, that doesn't stop you from contributing in this sector, so here's a few things you can do to help:
 
-## Help check/mod beatmaps!
+- Look through or search the [Pending Beatmaps](https://osu.ppy.sh/beatmapsets?m=-1&s=4) listing for any artists or songs you like (or just choose at random).
+- Play the beatmaps and leave feedback in threads.
+- Read the [Modding Guide](/wiki/Modding "Modding") and the [Ranking Criteria](/wiki/Ranking_Criteria "Ranking Criteria") to know the basics of technical modding and what makes a beatmap rankable.
+- Join the `#modhelp` and `#modreqs` on in-game chat and join in the conversation (or just follow along).
+- If you have the time and dedication, make a [Modding Queue](/community/forums/60) and have people start coming to you with requests.
 
-While we have set teams that handle the approval process, there's nothing stopping anyone from helping out. It starts as easy as posting in pending beatmap threads leaving your positive feedback or constructive criticism. When you get good, there's no reason you can't join up on one of the approval teams :).
+## Join some user-run projects and contribute to making them happen (or try to make them better if they're active)
```

Comment:
This could be shortened imho
something like "Help user-run [community-run?] projects related to osu!", maybe